zeebe/engine/src/main/java/io/camunda/zeebe/engine/processing/batchoperation/BatchOperationSuspendProcessor.java

```diff
@@ -81,7 +81,9 @@ public void processNewCommand(
       final TypedRecord<BatchOperationLifecycleManagementRecord> command) {
     final var request =
         new AuthorizationRequest(command, AuthorizationResourceType.BATCH, PermissionType.UPDATE);
-    final var authorizationResult = authCheckBehavior.isAuthorized(request);
+    final var authorizationResult =
+        authCheckBehavior.isAuthorized(
+            request, command.hasRequestMetadata(), command.getBatchOperationReference());
```

Comment:
🔧 These changes (not just here but in most processors) are very noisy and still don't communicate the intent clearly IMO.
I'd suggest one of these two alternatives instead:
```
if (authCheckBehavior.isExternalCommand(command) {
  authCheckBehavior.isAuthorized(new AuthorizationRequest(...));
}
```

or

```
authCheckBehavior.isAuthorizedOrInternalCommand(new AuthorizationRequest(...));
```